modules/test/playwright/tests/api-builder/apiFilter.spec.ts

```diff
@@ -0,0 +1,131 @@
+/**
```

Comment:
I don't know about the `api-builder` folder structure. We should probably create a different naming and structure. Something like including a subfolder for API testing within the Headless-builder folder

Reply:
Yeah, that is actually a good idea, @4lejandrito what do you think about it?

Reply:
@albertojml this is a test refactor so I just used the first LPS that was related to the fiunctionality :D

Reply:
I would put it inside `headless-builder-impl`, that's the module that is testing.